.github/workflows/conda-build-test.yml

```diff
@@ -0,0 +1,40 @@
+name: conda builds
+
+on:
+  pull_request:
+    paths:
+      - 'conda-environment.yml'
+  schedule:
+    - cron: '17 22 * * 6'
+  workflow_dispatch:
+
+jobs:
+  test_conda_install:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: checkout cylc-flow
+        uses: actions/checkout@v2
+
+      - name: build conda env
+        run: |
+            # The appended lines make sure that we actually build with Cylc
```

Comment:
@oliver-sanders 
This feels wierd.

Is it really necessary to check the `cylc version`, which makes all the crud here necessary?

Reply:
> This feels wierd.

It's fine (I do the same thing in the script I use to build my dev environments)

> Is it really necessary to check the cylc version.

Er, didn't you add this bit?

The point is more about checking the conda packages than checking the conda environment so it's not really necessary.